pkgs/top-level/all-packages.nix

```diff
@@ -12578,6 +12578,8 @@ with pkgs;
 
   react-native-debugger = callPackage ../development/tools/react-native-debugger { };
 
+  reactphysics3d = callPackage ../development/libraries/reactphysics3d { };
```

Comment:
Please move the package to `pkgs/by-name/re/reactphysics3d/package.nix` and remove this line, thanks.

Reply:
May i ask why? Seems like e nitpick since it is a library

Reply:
new top-level packages that use `pkgs.callPackage` should be in `by-name`

Reply:
> new top-level packages that use `pkgs.callPackage` should be in `by-name`

I see, so i should move it and remove it from top-level packages?